webapp/tests/karma/unit/services/resource-icon.js

```diff
@@ -179,7 +179,8 @@ describe('ResourceIcons service', function() {
           .to.equal(undefined);
 
         Changes.args[0][0].callback({ id: 'resources' }); // invoke the changes listener
-        service.replacePlaceholders(dom);
+        var bypassCache = true;
+        service.replacePlaceholders(dom, bypassCache);
```

Comment:
I don't think this test is correct as it doesn't simulate something that actually happens in the app: nothing calls `replacePlaceholders` with your new argument.
The text can be fixed, however, if you move the `replacePlaceholders` call inside the timeout callback, without using the `bypassCache` param.